getting_started/data_backup.md

```diff
@@ -0,0 +1,74 @@
+3. Enter the number associated with the type of remote (as of 11/8/21, Box was 6 and Google Drive was 15)
+4. Now the procedure bifurcates a bit...
+
+#### Google Drive
+5. You should have gotten a link to [this page](https://rclone.org/drive/#making-your-own-client-id). Follow the instructions to create a client id. One gotcha here is that **you won't be able to do this while logged into your andrew.cmu.edu** GSuite account, because you won't have permissions to create a project in that organization, so make sure to do it with a personal Google ID. (Don't worry, you'll still be able to link it to your Andrew GDrive space later) 
+6. You should eventually get a client ID and secret that you can paste into the `rclone` prompt to continue configuring. 
+7. Provide full access (option 1) at the `scope` prompt.
+8. Continue with default options until the question "Use auto config?" Because we are on a remote machine, you should enter `n` for this. 
+9. You will get a link at which you should authenticate with your Google account. **At this step, be sure to use your Andrew account if that's where you want to back up!"** You may get a warning, proceed anyway (in a Chrome browser, by clicking Advanced and then "go to rclone"). 
+10. After authenticating, copy the code and paste into the rclone prompt.
+11. From here, you should be able to continue with default options and finish configuring! Phew!
+
+#### Box
+5. Follow default options until the "use auto config?" prompt. Because we are on a remote machine, you should enter `n` for this. 
+6. You will then need to open a local terminal session to access your web browser for authentication. Follow the instructions to do so.
+7. After authentication, on your local terminal, you should get something to paste into your remote session. Make sure to copy everything between the arrows.
+8. Follow defaults to finish configuration.
+
+### Running an `rclone` backup
+You will probably want to create a folder on your remote (GDrive or Box) in which your backups will live. Let's suppose you've done that and it's called `arjuna_backup`, and that the name of your remote is `my_remote` (if you don't remember what you called it, run `rclone listremotes`).
+
+Assuming you are in your home directory and you want to back up the entire thing, the syntax would then be
+```bash
+rclone sync -P . my_remote:arjuna_backup
+```
+The `-P` flag isn't necessary, but it will give you progress updates. You may get a bunch of warnings about symlinks not being copied.
+
+Note that for your first backup, if you have a lot of stuff in your directory, it may take a long time to run, and could get interrupted if your internet connection cuts out. `tmux` is a useful tool to get around this and keep tasks running even if you disconnect (many good tutorial easily found on Google).
+
+### Scheduling backups
+An easy "set-it-and-forget-it" approach for backups is to schedule them using cron jobs. A simple setup for this can be achieved by:
```

Comment:
Add a default backup script and go full script kiddie here. Realistically, people are just going to copy-pasta, might as well make it easier for them

Reply:
maybe you want to write the canonical one? I just have one with a bunch of specific directories of mine lol